packages/dynamite/dynamite_runtime/lib/src/utils/codecs.dart

```diff
@@ -0,0 +1,112 @@
+// ignore_for_file: public_member_api_docs
```

Comment:
Huh, totally forgot to remove this and document the interface before submitting.